docs/manage_role_by_ldap_group.md

```diff
@@ -17,18 +17,23 @@ This guide provides instructions to manage roles by LDAP/AD group. You can impor
 
 Besides **[basic LDAP configure parameters](https://github.com/vmware/harbor/blob/master/docs/installation_guide.md#optional-parameters)** , LDAP group related configure parameters should be configured, they can be configured before or after installation
 
-  1. Configure parameters in harbor.cfg before installation
+  1. Configure LDAP parameters in with command line, please refer **[Config Harbor user settings by command line](configure_user_settings.md)**
```

Comment:
It should be `Configure LDAP parameters with command line`, no `in`, and it should be `refer to`.

Reply:
how about rephrase to `Configure LDAP parameters via API`

Reply:
OK